python_modules/libraries/dagster-dbt/dagster_dbt/asset_specs.py

```diff
@@ -60,6 +60,9 @@ def build_dbt_asset_specs(
         asset_out.to_spec(
             key=check.inst(asset_out.key, AssetKey),
             deps=[AssetDep(asset=dep) for dep in internal_asset_deps.get(output_name, set())],
+        )._replace(
+            skippable=False,
+            code_version=None,
```

Comment:
This potentially feels bad in a world where asset specs are fed into `@multi_asset` or `@dbt_assets`. They will not be able to compose.

Maybe this replacement should occur in the `external_assets_from_specs` function instead?

I don't think it's a big deal right now. We can optimize the composability/migration path from external assets to materializable assets later. Just calling this out.

Reply:
I don't think it would be very difficult to allow setting `code_version` for external assets.

Separately, kind of for this reason, I have been wondering whether `AssetsDefinition` should accept a `skippable_asset_keys` argument instead of having a `skippable` property on `AssetSpec`s.  Unlike the other properties on `AssetSpec`, `skippable` is a property of the computation, not the asset.